### ISSUE TITLE
Translate readme.md into english please

### ISSUE THREAD

**Issue by mWaltari** [winddriver/Delphi-Cross-Socket#48]
And also would be cool if English would be used as main communication language, since there are many international users around.

Currently projet is little bit hard to follow.

Keep up the good work!

**Comment by winddriver** [comment]
好的，我可以用google翻译一份，我的英文实在不怎么样，google翻译一定比我自己翻译得更好 :D

**Comment by mWaltari** [comment]
Google translate is an good start.

My English ain't perfect either. Still I bet your English is way better than my Chinese :D
